### ISSUE TITLE
Unable to install purescript

### ISSUE THREAD

**Issue by danielo515** [purescript/npm-installer#11]
Acording to official purescript docs, `npm i -g purescript` is the official way to install purescript. However it fails with a syntax error and I think it is because this package:

```
➜  npm i -g purescript
/Users/danielo/.nvm/versions/node/v8.15.0/bin/purs -> /Users/danielo/.nvm/versions/node/v8.15.0/lib/node_modules/purescript/purs.bin

> purescript@0.13.0 postinstall /Users/danielo/.nvm/versions/node/v8.15.0/lib/node_modules/purescript
> install-purescript --purs-ver=0.13.0

/Users/danielo/.nvm/versions/node/v8.15.0/lib/node_modules/purescript/node_modules/install-purescript-cli/index.js:97
	} catch {
	        ^

SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:617:28)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3
npm ERR! code ELIFECYCLE
```
I'm using node 8.15.0

Regards

**Comment by hdgarrood** [comment]
Yes, node 8 is not supported in the currently published version. Support for node 8 has been restored since then, though, so this will be fixed the next time we make a release. In the meantime you can fix this by upgrading to node 10+.

**Comment by danielo515** [comment]
I come back here to report that it works with node 11 and just saw your response @hdgarrood 
Thankfully I have nvm and I can switch between node versions, but most people just install the LTE version of node, which is node 8. When I saw the issue about adding back support for node 8 closed I took for granted that it was already deployed.

Seems that I'm cursed about purescript. Every time I try to come back to it something unexpected happens (last time was migration from 0.11 to 0.12)

Thanks for the response.
